web/src/helpers.tsx

```diff
@@ -130,3 +132,20 @@ export const timestampToLocaleString = (timestamp: number) => {
 
   return moment.unix(timestamp).format('YYYY-MM-DD HH:mm:ss');
 };
+
+/**
+ * Transform json string to yaml string
+ * @param jsonStr
+ */
+export const json2yaml = (jsonStr: string): string => {
+  return yaml.dump(JSON.parse(jsonStr))
```

Comment:
if jsonStr's value is `''`, an error will occur

Reply:
thanks a lot, already fixed.

use `try...catch` to handle the exception